subsys/lorawan/Kconfig

```diff
@@ -0,0 +1,58 @@
+# LoRaWAN configuration options
+
+# Copyright (c) 2019 Manivannan Sadhasivam <mani@kernel.org>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig LORAWAN
+	bool "LoRaWAN"
```

Comment:
I would recommened that you add `[EXPERIMENTAL]` until the API is a bit more stable. This way it is easier to change things in the API if needed.

Reply:
Okay